source/contributing.rst

```diff
@@ -1,22 +1,28 @@
 Contributing
 ============
 
-If you’re reading this you’re probably interested in contributing to one of the
-Hyper projects! First, we'd like to say thankyou! Projects like this one
+If you’re reading this, you’re probably interested in contributing to one of the
+Hyper projects! First, we'd like to say thank you! Projects like this one
 live-and-die based on the support they receive from others, and the fact that
 you’re even considering supporting the Hyper project is incredibly generous of
 you.
 
 This document lays out general guidelines and advice for contributing to any of
-the Hyper projects. If you’re thinking of contributing, start by reading this
-thoroughly and getting a feel for how contributing to the project works. If
+the Hyper projects. If you’re thinking of helping, start by reading this
+thoroughly and getting a feel for how the project works. If
 you’ve still got questions after reading this, please ask us in IRC: we'd be
 happy to clarify any questions you may have.
 
-The guide is split into sections based on the type of contribution you’re
-thinking of making, with a section that covers general guidelines for all
-contributors. Additionally, there is a section to cover our code of conduct,
-which applies to all contributions of any kind.
+We've organized this guide into sections:
+
+- our code of conduct, which applies to all contributions of any kind
+- general guidelines for all contributors
+- specific information based on the type of contribution you’re
+  thinking of making
+       - code contributions
+       - documentation
+       - bug reports
+       - feature requests
```

Comment:
This is fab. In the future we should make these internal links to the relevant sections, but that's not a reason to not merge this.